storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -73,6 +73,20 @@
     public static final String STORM_MESSAGING_NETTY_BUFFER_SIZE = "storm.messaging.netty.buffer_size";
 
     /**
+     * Netty based messaging: The write buffer high water mark for write buffer
```

Comment:
These config descriptions are not descriptive. Without reading the code, I have no idea what this does.